products/memgraph/upcoming/tutorials/got-deaths.md

```diff
@@ -0,0 +1,236 @@
+the Breaker of Chains or shortened to "Daenerys Targaryen" in our database is the biggest killer on
+the show. Let's list all the episodes she killed in as well as characters she killed.
+
+```opencypher
+MATCH (daenerys:Character {name: 'Daenerys Targaryen'})-[:KILLED]->(victim:Character)
+MATCH (daenerys)-[:KILLER_IN]->(d:Death)<-[:VICTIM_IN]-(victim)
+MATCH (d)-[:HAPPENED_IN]-(e:Episode)
+RETURN DISTINCT(victim.name) AS victim, COUNT(d) AS kill_count, e.name AS episode_name
+ORDER BY kill_count DESC
+```
+
+7) Remember that shocking last episode of the fifth season when they killed Jon Snow and we totally thought
+he was gonna stay dead? Well, let's list all the characters that would survive if he actually stayed dead.
+
+```opencypher
+MATCH (jon:Character {name: 'Jon Snow'})-[:KILLED]->(victim:Character)
+MATCH (jon)-[:VICTIM_IN]->(jon_death:Death)
+MATCH (jon)-[:KILLER_IN]->(victim_death:Death)<-[:VICTIM_IN]-(victim)
+WHERE victim_death.order>jon_death.order
+RETURN DISTINCT(victim.name) AS victim, COUNT(victim_death) AS kill_count
+ORDER BY kill_count DESC
+```
+
+8) Houses or allegiances are one of the main aspects of Westeros. Some houses killed more characters
+than others, but that doesn't matter in the end, what matters is efficiency. Let's list the allegiances with
+the best Kill/Death Ratios or KDR for short. Here we came across one additional problem. If an allegiance had more 
+deaths than kills, the KDR would be 0. This can easily be fixed with the `toFloat()` function.
+
+```opencypher
+MATCH (:Character)-[death:KILLED]->(:Character)-[:LOYAL_TO]->(a:Allegiance)
+WITH a, sum(death.count) as deaths
+MATCH (:Character)<-[kill:KILLED]-(:Character)-[:LOYAL_TO]->(a)
+RETURN a.name AS allegiance_name, SUM(kill.count) AS kills, deaths, ROUND(100 *(TOFLOAT(SUM(kill.count))/deaths))/100 AS KDR
+ORDER BY KDR DESC;
+```
+
+9) One of the best-rated episodes, Battle of the Bastards, showed us a fight between two houses: Stark and Bolton.
+Let's see which one had more casualties.
+
+```opencypher
+MATCH (c:Character)-[:LOYAL_TO]->(a:Allegiance)
+MATCH (c)-[:VICTIM_IN]-(d:Death)-[:HAPPENED_IN]-(:Episode {name: 'Battle of the Bastards'})
+RETURN a.name AS house_name, COUNT(d) AS death_count
+ORDER BY death_count DESC
+LIMIT 2;
+```
+
+10) Who do you think was the biggest traitor in terms of killing in its own allegiance? Well, let's check it out!
+
+```opencypher
+MATCH (killer:Character)-[:KILLED]->(victim:Character)
+MATCH (killer)-[:LOYAL_TO]->(a:Allegiance)<-[:LOYAL_TO]-(victim)
+RETURN killer.name AS traitor, COUNT(victim) AS kill_count
+ORDER BY kill_count DESC
+```
+
+11) One of the biggest features of Memgraph is drawing the graphs of queries we execute. Let's visualize all the
+Loyalties with Characters. Execute the following query and head out to the `GRAPH` tab.
+
+```opencypher
+MATCH (c:Character)-[i:LOYAL_TO]-(l) RETURN c,i,l;

```

Comment:
You could benefit from having a path variable here when you want to visualize. Check other tutorials for help (e.g. football transfers).